src/languages/snowflake/snowflake.functions.ts

```diff
@@ -0,0 +1,614 @@
+    'BOOLAND',
+    'BOOLAND_AGG',
+    'BOOLNOT',
+    'BOOLOR',
+    'BOOLOR_AGG',
+    'BOOLXOR',
+    'BOOLXOR_AGG',
+    'BUILD_SCOPED_FILE_URL',
+    'BUILD_STAGE_FILE_URL',
+    'CASE',
+    'CAST',
+    '::', // this is an infix operator
+    'CBRT',
+    'CEIL',
+    'CHARINDEX',
+    'CHECK_JSON',
+    'CHECK_XML',
+    'CHR',
+    'CHAR',
+    'COALESCE',
+    'COLLATE',
+    'COLLATION',
+    'COMPLETE_TASK_GRAPHS',
+    'COMPRESS',
+    'CONCAT',
+    '||', // this is an operator like "a" || "b"
+    'CONCAT_WS',
+    'CONDITIONAL_CHANGE_EVENT',
+    'CONDITIONAL_TRUE_EVENT',
+    'CONTAINS',
+    'CONVERT_TIMEZONE',
+    'COPY_HISTORY',
+    'CORR',
+    'COS',
+    'COSH',
+    'COT',
+    'COUNT',
+    'COUNT_IF',
+    'COVAR_POP',
+    'COVAR_SAMP',
+    'CUME_DIST',
+    'CURRENT_ACCOUNT',
+    'CURRENT_AVAILABLE_ROLES',
+    'CURRENT_CLIENT',
+    'CURRENT_DATABASE',
+    'CURRENT_DATE',
+    'CURRENT_IP_ADDRESS',
+    'CURRENT_REGION',
+    'CURRENT_ROLE',
+    'CURRENT_SCHEMA',
+    'CURRENT_SCHEMAS',
+    'CURRENT_SECONDARY_ROLES',
+    'CURRENT_SESSION',
+    'CURRENT_STATEMENT',
+    'CURRENT_TASK_GRAPHS',
+    'CURRENT_TIME',
+    'CURRENT_TIMESTAMP',
+    'CURRENT_TRANSACTION',
+    'CURRENT_USER',
+    'CURRENT_VERSION',
+    'CURRENT_WAREHOUSE',
+    'DATA_TRANSFER_HISTORY',
+    'DATABASE_REFRESH_HISTORY',
+    'DATABASE_REFRESH_PROGRESS',
+    'DATABASE_REFRESH_PROGRESS_BY_JOB',
+    'DATABASE_STORAGE_USAGE_HISTORY',
+    'DATE_FROM_PARTS',
+    'DATE_PART',
+    'DATE_TRUNC',
+    'DATEADD',
+    'DATEDIFF',
+    'DAYNAME',
+    'DECODE',
+    'DECOMPRESS_BINARY',
+    'DECOMPRESS_STRING',
+    'DECRYPT',
+    'DECRYPT_RAW',
+    'DEGREES',
+    'DENSE_RANK',
+    'DIV0',
+    'EDITDISTANCE',
+    'ENCRYPT',
+    'ENCRYPT_RAW',
+    'ENDSWITH',
+    'EQUAL_NULL',
+    'EXP',
+    'EXPLAIN_JSON',
+    'EXTERNAL_FUNCTIONS_HISTORY',
+    'EXTERNAL_TABLE_FILES',
+    'EXTERNAL_TABLE_FILE_REGISTRATION_HISTORY',
+    'EXTRACT',
+    'EXTRACT_SEMANTIC_CATEGORIES',
+    'FACTORIAL',
+    'FIRST_VALUE',
+    'FLATTEN',
+    'FLOOR',
+    'GENERATE_COLUMN_DESCRIPTION',
+    'GENERATOR',
+    'GET',
+    'GET_ABSOLUTE_PATH',
+    'GET_DDL',
+    'GET_IGNORE_CASE',
+    'GET_OBJECT_REFERENCES',
+    'GET_PATH',
+    ':', // this is an infix operator AS WELL as a function
+    'GET_PRESIGNED_URL',
+    'GET_RELATIVE_PATH',
+    'GET_STAGE_LOCATION',
+    'GETBIT',
+    'GREATEST',
+    'GROUPING',
+    'GROUPING_ID',
+    'HASH',
+    'HASH_AGG',
+    'HAVERSINE',
+    'HEX_DECODE_BINARY',
+    'HEX_DECODE_STRING',
+    'HEX_ENCODE',
+    'HLL',
+    'HLL_ACCUMULATE',
+    'HLL_COMBINE',
+    'HLL_ESTIMATE',
+    'HLL_EXPORT',
+    'HLL_IMPORT',
+    'HOUR',
+    'MINUTE',
+    'SECOND',
+    'IFF',
+    'IFNULL',
+    'ILIKE',
+    'ILIKE ANY',
+    '[ NOT ] IN',
+    'INFER_SCHEMA',
+    'INITCAP',
+    'INSERT',
+    'INVOKER_ROLE',
+    'INVOKER_SHARE',
+    'IS [ NOT ] DISTINCT FROM',
+    'IS [ NOT ] NULL',
```

Comment:
This `[ ... ]` syntax won't work in here because this list of strings is not wrapped inside `expandPhrases()` call.

Besides, these are again operators, not function names. That it, you won't be calling them as `IS NOT NULL(value)`